engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version.html

```diff
@@ -25,7 +25,7 @@
     <li ng-class="{active: isState('content-view.version.package-groups')}"
         ng-show="version.package_group_count !== 0">
       <a ui-sref="content-view.version.package-groups({versionId: version.id})">
-        <span translate>rpm Package Groups</span>
+        <span translate>rpm Package Groups (Deprecated)</span>
```

Comment:
I'm not sure we're removing package groups across the board ?  Just the host actions, right?